cabal.project

```diff
@@ -17,8 +17,9 @@ repository cardano-haskell-packages
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-executable-spec.git
-  tag: d6effa703ed06ffc6239a963fdbccccd25dfa663
-  --sha256: sha256-b+HRjMrriceFM/fUn9Qn+LJjMh3I3/F9PtYxrmGOiug=
+  -- branch: aniketd/govcert
```

Comment:
This is required for `nix develop` to work, but required to be absent for `cabal build`, as far as I can tell.

Reply:
Well, apparently not :+1: